### PR TITLE
cms.util.mkdir sets owner and permission of new directory

### PR DESCRIPTION
**Pull request by akmohtashami**
A solution for #579

[Review on Reviewable](https://reviewable.io/reviews/cms-dev/cms/580)

### CODE REVIEW

File: cms/util.py [hunk 1]
```diff
@@ -29,6 +29,8 @@
 import netifaces
 import os
 import sys
+import pwd
```
Comment (stefano-maggiolo) [inline cms/util.py]: Codacy Issue found: [Unused import pwd](https://www.codacy.com/app/s-maggiolo/cms/file/2193768415/issues/source?bid=3167167&fileBranchId=3167369#l32)

Comment (akmohtashami) [inline cms/util.py]: Fixed.

**Comment by stefano-maggiolo** [comment]
Marged, thank you - only changed the raise to a return False as stated in the pydoc.

**Comment by stefano-maggiolo** [comment]
If you want to send a PR to add yourself in the AUTHOR.txt file, please do!

**Pull request closed without merging**